[Content Updates] Improvements for more docs (#7063)
flutter/website: src/get-started/test-drive/_vscode.md

## Run the app

1. Locate the VS Code status bar
   (the blue bar at the bottom of the window):<br>
   ![status bar][]{:.mw-100.pt-1.pb-2}
1. Select a device from the **Device Selector** area.
   For details, see [Quickly switching between Flutter devices][].
   - If no device is available, and you want to use a device simulator,
     click **No Devices** and click
     **Start iOS Simulator** to launch a simulator.

     {{site.alert.warning}}

>        You may not see **Start iOS Simulator** option

Reviewer: ```suggestion
       You might not see **Start iOS Simulator** option
```

       when you click **No Devices** in VS Code.

>        If you are on Mac then you may have to run following command

Reviewer: ```suggestion
       If you're on Mac, then you might have to run following command
```

       in terminal to launch a simulator.

       ```terminal
       $ open -a simulator
       ```

>        On Windows or Linux it is not possible to launch iOS simulator.

Reviewer: ```suggestion
       On Windows or Linux, it's not possible to launch an iOS simulator.
```

     {{site.alert.end}}

   - To setup a real device, follow the device-specific instructions
     on the [Install][] page for your OS.
1. Invoke **Run > Start Debugging** or press <kbd>F5</kbd>.

> 1. Wait for the app to launch &mdash; progress is printed

Reviewer: ```suggestion
1. Wait for the app to launch&mdash;progress is printed
```
